redis/howto/migratefromregionaltoglobal.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Migrate Regional to Global Database
+---
+
+This guide will help you migrate your data from a regional Upstash Redis database to a global database.
+We strongly recommend you to use [Upstash Redis Global](https://upstash.com/docs/common/concepts/global-replication) for your application.
+Our legacy regional Redis databases will be deprecated in the future. Upstash Global Redis has better performance and more feature set
+
+## Prerequisites
+
+Before starting the migration, make sure you have:
+
+1. An existing regional Upstash Redis database (source)
+2. A new global Upstash Redis database (destination)
+3. Access to both databases' credentials (connection strings, passwords)
+
+## Migration Process
+
+There are several official ways to migrate your data:
+
+### 1. Using Backup/Restore (Recommended for AWS Regional Databases)
+
+If your regional database is hosted in AWS, you can use Upstash's backup/restore feature:
+
+1. Create a backup of your regional database:
+   - Go to your regional database details page
+   - Navigate to the `Backups` tab
+   - Click the `Backup` button
+   - Provide a unique name for your backup
+   - Wait for the backup process to complete
+
+   <Info>
+     During backup creation, some database operations will be temporarily unavailable.
+   </Info>
+
+2. Restore the backup to your global database:
+   - Go to your global database details page
+   - Navigate to the `Backups` tab
+   - Click `Restore...`
+   - Select your regional database as the source
+   - Select the backup you created
+   - Click `Start Restore`
+
+   <Warning>
+     The restore operation will flush (delete) all existing data in your global database before restoring the backup.
+   </Warning>
+
+### 2. Using Upstash Console Migration Wizard
+
+The easiest way to migrate your data is using the Upstash Console's built-in migration wizard:
+
+1. Go to [Upstash Console](https://console.upstash.com)
+2. In the database list page, click the `Import` button
+3. Select your source (regional) database
+4. Select your destination (global) database
+5. Follow the wizard instructions to complete the migration
+
+<Info>
+  Note: The destination database will be flushed before migration starts.
+</Info>
+
+### 3. Using upstash-redis-dump
+
+Another reliable method is using the official [upstash-redis-dump](https://github.com/upstash/upstash-redis-dump) tool:
+
+1. Install upstash-redis-dump:
+   ```bash
+   npm install -g upstash-redis-dump
+   ```
+
+2. Export data from regional database:
+   ```bash
+   upstash-redis-dump -db 0 -host YOUR_REGIONAL_HOST -port 6379 -pass YOUR_PASSWORD -tls > redis.dump
```

Comment:
regional database's port may not be 6379 and tls may be disabled

Reply:
thanks updated